### PR TITLE
✨ New Extension: CodeBlocks

### PR DESCRIPTION
**Pull request by tisfeng**
## Description

Set or modify the programming language in markdown code blocks in batch.

If you are a developer and often write markdown code blocks, you may need it.

## Screencast

CodeBlocks

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by danielsequeira** [comment]
Hi @tisfeng 👋 

Thanks for your pull request.

Could you please instead of the `.gif` file, add some screenshots of your extension to the `metadata` folder? As mentioned here: https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots

**Comment by tisfeng** [comment]
No problem, but I want to ask, is gif not allowed? Because in my opinion, gif is also a kind of screenshot picture. For my extension, gif can better show its feature.
image

**Comment by danielsequeira** [comment]
@tisfeng we don't really mention it in the docs (we will update that), but considering how our store is implemented, we would prefer if you could add a few screenshots in png or jpg format.

**Comment by tisfeng** [comment]
Ok, I got it.

**Comment by tisfeng** [comment]
@danielsequeira  I have changed it. Finally, I still want to know, will your team consider supporting such dynamic display like GIF in the future? Because some extension use GIF to show the features is really better than static images.

**Comment by danielsequeira** [comment]
> @danielsequeira I have changed it. Finally, I still want to know, will your team consider supporting such dynamic display like GIF in the future? Because some extension use GIF to show the features is really better than static images.

Thanks @tisfeng, appreciate it. We started this discussion internally, I don't have an answer for you right now regarding it, but definitely something that we are considering.

**Comment by tisfeng** [comment]
@danielsequeira I have updated the first screenshot, thank you for your careful discovery.

File: extensions/CodeBlocks/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  "categories": [
+    "Developer Tools",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Set Language",
+      "subtitle": "Code Blocks",
```
Comment (danielsequeira) [inline extensions/CodeBlocks/package.json:17]: Sorry for the nitpicking, but just noticed this. In order to be coherent with the extension name, it should be CodeBlocks right?
```suggestion
      "subtitle": "CodeBlocks",
```

**Comment by danielsequeira** [comment]
@tisfeng my pleasure, just 2 more small things and we are ready to merge

File: extensions/CodeBlocks/package.json (hunk 1 above)
Comment (tisfeng) [reply extensions/CodeBlocks/package.json]: Yeah, thank you @danielsequeira

**Review by danielsequeira — APPROVED** [review]

**Pull request merged**